bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/InstarHandler.java

```diff
@@ -62,43 +63,61 @@ public void channelRead(@Nullable ChannelHandlerContext ctx, @Nullable Object ms
             ipCameraHandler.logger.trace("HTTP Result back from camera is \t:{}:", content);
             switch (requestUrl) {
                 case "/param.cgi?cmd=getinfrared":
-                    if (content.contains("var infraredstat=\"auto")) {
+                    if (content.contains("var infraredstat=\"auto") || content.contains("infraredstat=\"2")) {
```

Comment:
Isn't a final `\"` missing to match value 2?

Reply:
Agree safer in case api gets changed. Made change so vale 22, 222 etc do not match